climate-explorer/templates/monit_config.j2

```diff
@@ -4,6 +4,6 @@ check host climate_explorer_{{ user }} with address 127.0.0.1
     if failed
         port {{ climate_explorer_port }}
         protocol http
-        request "/v1/projects"
+        request "v1/activities?page=1&limit=3"
```

Comment:
Did you mean to remove the first slash here? And if so, do you need to remove the first slash from the other URL path?

Reply:
Another good save by @Starttoaster

Reply:
To be clear, I have absolutely no clue. Resolve and call me an idiot  if it doesn't matter